<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GamepadButtonIconSelectionDialog</class>
 <widget class="QWidget" name="GamepadButtonIconSelectionDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="tTitleLabel" name="titleLabel">
     <property name="text">
      <string>Icon Type</string>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_2">
     <item>
      <widget class="QListWidget" name="optionsWidget">
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="GamepadHud" name="gamepadHud" native="true"/>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GamepadHud</class>
   <extends>QWidget</extends>
   <header location="global">gamepadhud.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>tTitleLabel</class>
   <extends>QLabel</extends>
   <header location="global">ttitlelabel.h</header>
   <slots>
    <signal>backButtonClicked()</signal>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
